lib/open_mic.rb

```diff
@@ -0,0 +1,18 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+  def initialize(info_hash)
+    @location = info_hash[:location]
+    @date = info_hash[:date]
+    @performers = []
+  end
+
+  def welcome(performer)
+    @performers << performer
+  end
+
+  def repeated_jokes?
+    @performers[1].jokes.include?(@performers[0].jokes[0])
+    # require "pry"; binding.pry
```

Comment:
so... technically this makes your test pass. however, this will only work as long as the performers are exactly two people, and the first person knows exactly two jokes, and the second person knows exactly the same joke as the first performer's first joke.  Basically... you wrote this method to fit exactly one use case, which is not what we would want. 

to know if there are any duplicate jokes, you'd first need to know all of the jokes

```
all_jokes = @performers.map do |user|
    user.jokes
end
```

Then, check if the unique list of those jokes matches the original list of those jokes

`all_jokes.flatten.uniq.length != all_jokes.flatten.length`

we use flatten, because all jokes is actually an array with arrays inside. `flatten` gets rid of the inner arrays.